gateway-service-hashicorp-vault/src/test/java/org/apache/knox/gateway/backend/hashicorp/vault/TestHashicorpVaultAliasService.java

```diff
@@ -184,6 +185,9 @@ public void testVaultIntegration() throws Exception {
     assertNull(aliasService.getPasswordFromAliasForCluster(clusterName, alias));
     assertEquals(0, aliasService.getAliasesForCluster(clusterName).size());
 
+    char[] generatedPassword = aliasService.getPasswordFromAliasForCluster(clusterName, alias, true);
+    assertNotNull(generatedPassword != null);
+
```

Comment:
You may want to make sure the generated password for `alias` is not the same as `aliasPassword`

Reply:
Thanks!